@dgault @sbesson I had to exclude ``edu.ucar:cdm-core`` declared in https://github.com/ome/bioformats/blob/develop/components/formats-gpl/pom.xml#L67
The build fails with the error:
```
[WARNING] The POM for edu.ucar:cdm-core:jar:5.3.3 is missing, no dependency information available
Downloading from ome.maven: https://artifacts.openmicroscopy.org/artifactory/maven/edu/ucar/cdm-core/5.3.3/cdm-core-5.3.3.jar
Downloading from central: https://repo.maven.apache.org/maven2/edu/ucar/cdm-core/5.3.3/cdm-core-5.3.3.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.468 s
[INFO] Finished at: 2022-12-15T20:17:55Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project downloader: Could not resolve dependencies for project omero:downloader:jar:0.2.3-SNAPSHOT: Could not find artifact edu.ucar:cdm-core:jar:5.3.3 in ome.maven (https://artifacts.openmicroscopy.org/artifactory/maven) -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -42,12 +42,6 @@
         <groupId>org.openmicroscopy</groupId>
         <artifactId>omero-gateway</artifactId>
         <version>5.7.0</version>
-         <exclusions>
-        <exclusion>
-          <groupId>edu.ucar</groupId>
-          <artifactId>cdm-core</artifactId>
-        </exclusion>
-      </exclusions>
       </dependency>
       <dependency>
         <groupId>commons-cli</groupId>
@@ -100,12 +94,16 @@
 
   <repositories>
       <repository>
+        <id>unidata.releases</id>
+        <url>https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases</url>
+      <snapshots><enabled>false</enabled></snapshots>
+      </repository>
+      <repository>
         <id>ome.maven</id>
         <name>OME Maven Artifactory</name>
         <url>https://artifacts.openmicroscopy.org/artifactory/maven</url>
       </repository>
   </repositories>
-
   <build>
     <defaultGoal>package</defaultGoal>
     <plugins>

